community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -79,7 +79,7 @@ public ReadOperations readOperations()
         if( !transaction.mode().allowsReads() )
         {
             throw new AuthorizationViolationException(
-                    String.format( "Read operations are not allowed for `%s` transactions.", transaction.mode().name() ) );
+                    String.format( "Read operations are not allowed for '%s'.", transaction.mode().name() ) );
```

Comment:
I also checked, and the back-ticks look like back-ticks in the browser, so no markdown support.